src/abft-messages-bundles.md

```diff
@@ -0,0 +1,45 @@
+# Bundles
+
+Let \\(V\\) be any set of votes and equivocation votes.
+
+We say that \\(V\\) _is a bundle for \\(v\\) in round \\(r\\), period \\(p\\), and
+step_ \\(s\\) (or a _bundle for \\(v\\) at_ \\((r, p, s)\\)), denoted \\(Bundle(r, p, s, v)\\).
+
+> ⚙️ **IMPLEMENTATION**
+>
+> Bundle [reference implementation](https://github.com/algorand/go-algorand/blob/b6e5bcadf0ad3861d4805c51cbf3f695c38a93b7/agreement/bundle.go#L46).
```

Comment:
Making links like this is going to be hell to maintain.

Reply:
We are using permalinks. Although this does not guarantee automatic updates, it pins the code to a specific commit. I think is valuable to have direct references to the implementation.

Reply:
We can make it clear in the reading guidelines.

Reply:
#121